operate/ledger/__init__.py

```diff
@@ -46,17 +46,6 @@
 POLYGON_RPC = os.environ.get("POLYGON_RPC", "https://polygon-rpc.com")
 SOLANA_RPC = os.environ.get("SOLANA_RPC", "https://api.mainnet-beta.solana.com")
 
-PUBLIC_RPCS = {
-    Chain.ARBITRUM_ONE: ARBITRUM_ONE_PUBLIC_RPC,
-    Chain.BASE: BASE_PUBLIC_RPC,
-    Chain.CELO: CELO_PUBLIC_RPC,
-    Chain.ETHEREUM: ETHEREUM_PUBLIC_RPC,
-    Chain.GNOSIS: GNOSIS_PUBLIC_RPC,
-    Chain.MODE: MODE_PUBLIC_RPC,
-    Chain.OPTIMISM: OPTIMISM_PUBLIC_RPC,
-    Chain.POLYGON: POLYGON_PUBLIC_RPC,
-    Chain.SOLANA: SOLANA_PUBLIC_RPC,
-}
```

Comment:
This dictionarly doesn't make much sense. The public endpoints should be the default value if the env var is not specified. We have now a single dictionary DEFAULT_RPCs which is populated with the environment variables OR the public RPC hardcoded value.